index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <nav id="navbar">
+      <a href="#projects">Projects</a>
+    </nav>
```

Comment:
I think the nav tag would be better wrapped inside a header tag.
Like this;
`<header>
    <nav></nav>
</header>`
This is important for better semantics and to help on screen readers identify what part of the code is being read.

Reply:
Well done using inline comments 🎉 but either this is not correctly formatted or something went wrong because I'm not able to see the example code. I guess it should've been like this:
```
example code
```
Yours is appearing like this:
![Screenshot 2023-02-20 at 21 41 38](https://user-images.githubusercontent.com/45739963/220233916-32a6eb76-2770-4dc0-a52e-75e36e876614.png)